docs/metrics-via-grafana.md

```diff
@@ -0,0 +1,73 @@
+# Setup Prometheus and Grafana
+
+## Steps
+
+### 1. Create `prometheus.yaml` config file
+
+This is an example:
+```
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "Test Atlas Operator Project-mongo-metrics"
+    scrape_interval: 10s
+    metrics_path: /metrics
+    scheme : https
+    basic_auth:
+      username: user
```

Comment:
not important - probably better show as placeholder

Reply:
Ok, great catch 👍